cmd/umoci/main.go

```diff
@@ -65,14 +66,34 @@ func main() {
 
 	app.Flags = []cli.Flag{
 		cli.BoolFlag{
-			Name:  "debug",
-			Usage: "set log level to debug",
+			Name:  "verbose",
+			Usage: "alias for --log=info",
+		},
+		cli.StringFlag{
+			Name:  "log",
+			Usage: "set the log level (debug, info, [warn], error, fatal)",
+			Value: "warn",
 		},
 	}
 
 	app.Before = func(ctx *cli.Context) error {
-		if ctx.GlobalBool("debug") {
-			logrus.SetLevel(logrus.DebugLevel)
+		log.SetHandler(logcli.New(os.Stderr))
+
+		if ctx.GlobalBool("verbose") {
+			if ctx.GlobalIsSet("log") {
```

Comment:
I'd prefer to make it so people don't mix the two at all. But thanks for the review.